components/telemetry-controller/Makefile

```diff
@@ -0,0 +1,159 @@
+# More info on the usage of ANSI control characters for terminal formatting:
+# https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_parameters
+# More info on the awk command:
+# http://linuxcommand.org/lc3_adv_awk.php
+
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+##@ Development
+
+# Generate manifests e.g. CRD, RBAC etc.
+manifests-local: controller-gen-local
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+
+## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
+manifests: controller-gen
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+
+ ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
+generate: controller-gen
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+
+resolve-local:
+	GO111MODULE=on go mod vendor -v
+
+ensure-local:
+	@echo "Go modules present in component - omitting."
+
+dep-status:
+	@echo "Go modules present in component - omitting."
+
+dep-status-local:
+	@echo "Go modules present in component - omitting."
+
+mod-verify-local:
+	GO111MODULE=on go mod verify
+
+ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
+ ## Run tests.
+test-local: manifests-local generate-local fmt-local vet-local
+	mkdir -p ${ENVTEST_ASSETS_DIR}
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.8.3/hack/setup-envtest.sh
+	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
+
+ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
+ ## Run tests.
+test: manifests generate fmt vet
+	mkdir -p ${ENVTEST_ASSETS_DIR}
+	test -f ${ENVTEST_ASSETS_DIR}/setup-envtest.sh || curl -sSLo ${ENVTEST_ASSETS_DIR}/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.8.3/hack/setup-envtest.sh
+	source ${ENVTEST_ASSETS_DIR}/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
+
+##@ Build
+
+build: generate fmt vet ## Build manager binary.
+	go build -o bin/manager main.go
+
+run-local: manifests-local generate-local fmt-local vet-local ## Run a controller from your host.
+	go run ./main.go
+
+##@ Deployment
+
+## Will be called from Prow-Pipeline; using targets from generic make file
+release: resolve generate verify build-image push-image copy-external-crds
+
+install-local: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
```

Comment:
Local targets using `kustomize` fail, because `kustomize` binary is copied to `common/makefiles/bin` and is expected to be in `components/telemetry-collector/bin`

Reply:
Fixed by expecting existing kustomize binary (similar to eventing-controller)